doc/lspconfig.txt

```diff
@@ -278,13 +277,8 @@ where the structure of the docs table is as follows:
       default_config = {'t', true},
     }
 <
-`commands` is a map of `name:definition` key:value pairs, where `definition`
-is a list whose first value is a function implementing the command, and the
-rest are either array values which will be formed into flags for the command,
-or special keys like `description`.
 
-Warning: Commands is deprecated and will be removed in future releases.
-It is recommended to use `vim.api.nvim_create_user_command()` instead in an `on_attach` function.
+Custom commands only work for servers
```

Comment:
```suggestion
Custom commands only works for servers
```
I think that entire section about custom commands is not needed anymore?

Reply:
I think it can help some people so keep it . like some new vimer this can give them a tip

Reply:
I doubt new people are going to define custom commands. This is an advanced thing to do, and for that they should refer to neovim LSP documentation.

Reply:
got it. remove